ddp-workspace/projects/ddp-fon/app.yaml

```diff
@@ -1,5 +1,5 @@
 service: fon
```

Comment:
not active

Reply:
I replaced runtime version in all `app.yaml` for all studies found in `ddp/angular/projects` intentionally. Change doesn't hurt to have and we might need old study back in future. I don't have a strong opinion, let me know if you still want me to undo changes for inactive study.

Reply:
ok with change as long as we are not deploying.